apps/form-builder/src/components/NewFormQuestion/NewFormQuestion.tsx

```diff
@@ -0,0 +1,34 @@
+'use client'
+import { Button, Checkbox, FormControl, FormControlLabel, FormLabel, Radio, RadioGroup, TextField } from "@mui/material"
+
+export default function NewFormQuestion() {
+    return (
+        <div style={{display: 'flex', flexDirection: 'column'}}>
+            <TextField
+                id="formId" 
+                label="Form ID" 
+                required
+            />
+            <TextField 
+                id="title" 
+                label="Title" 
```

Comment:
great use of the `required` prop in other fields. could you add it to the title as well?